src/SignalR/clients/csharp/Http.Connections.Client/src/HttpConnection.Log.cs

```diff
@@ -66,6 +66,9 @@ private static class Log
             private static readonly Action<ILogger, HttpTransportType, Exception> _transportStarted =
                 LoggerMessage.Define<HttpTransportType>(LogLevel.Debug, new EventId(18, "TransportStarted"), "Transport '{Transport}' started.");
 
+            private static readonly Action<ILogger, Exception> _serverSentEventsNotSupportedByBrowser =
+                LoggerMessage.Define(LogLevel.Debug, new EventId(19, "ServerSentEventsNotSupportedByBrowser"), "Skipping ServerSentEvents because they are not supported by the browser.");
```

Comment:
```suggestion
                LoggerMessage.Define(LogLevel.Debug, new EventId(19, "ServerSentEventsNotSupportedByBrowser"), "Skipping ServerSentEvents because they it is not supported by the browser.");
```

Reply:
Should I change https://github.com/dotnet/aspnetcore/pull/20115/files#diff-0d9edc1e2ea2a38f9a4be7f30733cc9eR61 as well?

Reply:
Don't know what line you're trying to point out

Reply:
Whoops

https://github.com/dotnet/aspnetcore/blob/9908f8ab716e11f3165e2666c40fffb59646d294/src/SignalR/clients/csharp/Http.Connections.Client/src/HttpConnection.Log.cs#L61

Reply:
Hmm, we can just keep both as is. Not too picky

Reply:
The difference is that WebSockets is the name of a transport and a name for the type of socket. In that latter sense, that could be plural. ServerSentEvents is just the name of a transport, so it should probably be singular. Super nitpicky though.